README.md

```diff
@@ -47,6 +47,25 @@ All [configuration variables](https://github.com/eccenca/gui-elements/blob/devel
 
 ## Development
 
+### Branch management
+
+We have 4 types of major branches representing the current state:
+
+-   `main`: contains the latest official release, only release branches will be merged in this branch
```

Comment:
`in` -> `into`
Also, what are release branches? They do not seem to be described in the rest of the README even though the `release/` prefix is configured as allowed prefix.